packages/ui/src/lib/NativeBase/Navbar/index.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import React from "react";
+
+import NativeBase from "..";
+import Navbar from ".";
+
+export default {
+  // https://github.com/storybookjs/storybook/issues/16038
+  title: "@utofun ui/Navbar",
+  component: Navbar,
+  argTypes: {},
+} as ComponentMeta<typeof Navbar>;
+
+const Template: ComponentStory<typeof Navbar> = function Template(args) {
+  return (
+    <NativeBase>
+      <Navbar {...args} />
+    </NativeBase>
+  );
+};
+
+export const Utofun = Template.bind({});
```

Comment:
Navbar > Utofun?

Reply:
I concur. Maybe we can leave this to Intern?

Reply:
I'll do it myself next friday afternoon

Reply:
Ticket https://app.asana.com/0/1170389096525212/1202281013533417/f